scipy/spatial/ckdtree/src/ckdtree_decl.h

```diff
@@ -2,7 +2,7 @@
 #define CKDTREE_CPP_DECL
 
 /*
- * Use numpy to provide some platform independency.
+ * Use numpy to provide some platform independence.
```

Comment:
FWIW this is still a valid word, so no need to change it. But also no harm in doing so, it means the same thing as independence.